src/component/form/CaseElement.tsx

```diff
@@ -89,19 +105,65 @@ export const getExitName = (
     // Don't reassign func params
     let newExitName = exitName;
 
-    // Some operators don't expect args
-    if (newArgList.length >= 0 && !operatorConfig.categoryName) {
+    if (newArgList.length > 0 && !operatorConfig.categoryName) {
         newExitName = composeExitName(operatorConfig.type, newArgList);
-    } else {
+    } else if (operatorConfig.categoryName) {
+        // Some operators don't expect args
         // Use the operator's default category name
         ({ categoryName: newExitName } = operatorConfig);
     }
 
     return newExitName;
 };
 
-export const hasArgs = (args: string[] = []): boolean =>
-    args.length > 0 && args[0].trim().length > 0;
+/**
+ * Returns min, max values for 'has_number_between' case
+ */
+export const getMinMax = (args: string[] = []): { min: string; max: string } => {
+    let min = '';
+    let max = '';
+
+    if (args.length) {
+        if (strContainsNum(args[0])) {
+            min = args[0];
+        }
+
+        if (args[1]) {
+            if (strContainsNum(args[1])) {
+                max = args[1];
+            }
+        }
+    }
+
+    return {
+        min,
+        max
+    };
+};
+
+export const isFloat = (val: string): boolean => /^-?\d+(?:[.,]\d*?)?$/.test(val.trim());
```

Comment:
helpers to identify floats and ints, test strings for numbers and parse numbers from strings.